_guides/analytics/use_cases.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: page
+title: Use Cases
+order: 2
+---
+
+This guide provides a set of common use cases for tracking user engagement:
+
+{% include toc.html %}
+
+Want to add a use case? 
```

Comment:
Added one issue to track for now. Hope this is OK-- I can add a per vendor issue as well: https://github.com/ampproject/amphtml/issues/2258